frontend/src/components/EmployeeModal/EmployeeInfo.tsx

```diff
@@ -29,6 +29,7 @@ const EmployeeInfo = ({
           type="text"
           defaultValue={firstName}
           className={cn(styles.input)}
+          aria-required={true}
```

Comment:
```suggestion
          aria-required="true"
```

A light suggestion: use a string "true" instead of a JavaScript boolean. You're passing a pretty standard HTML parameter, so let's make the JSX match that instead of starting to mix types unnecessarily.

Reply:
yup sounds good. thanks for the suggestion!